democracy_club/apps/hermes/feeds.py

```diff
@@ -35,10 +35,13 @@ def item_categories(self, item):
         return [category.title for category in item.category.hierarchy()]
 
     def item_author_name(self, item):
-        return "{first_name} {last_name}".format(
-            first_name=item.author.first_name,
-            last_name=item.author.last_name,
-        )
-
+        authors = [author.name for author in item.author.all()]
+        if len(authors) == 1:
+            return authors[0]
+        elif len(authors) == 2:
+            return "{} {}".format(authors[0], authors[1])
+        elif len(authors) > 2:
+            return "{} {}".format(authors[0], authors[-1])
```

Comment:
I think this will just return the first and last authors in the case of >2?

This whole block can be simplified down to `", ".join(authors)` I think...as with one it'll just return the name, but with more than one it'll join them with a comma. I don't think we need the "and" on the last author in the feed, so that makes things a little simpler.

Reply:
I don't actually think we need this at all. It's already handled in the template.